modules/nuxt-basic-auth.yml

```
@@ -0,0 +1,16 @@
+name: nuxt-basic-auth
+description: Nuxt 3 Module for Basic Authentication
+repo: kgierke/nuxt-basic-auth
+npm: '@kgierke/nuxt-basic-auth'
+icon: ''
+github: https://github.com/kgierke/nuxt-basic-auth
+website: https://github.com/kgierke/nuxt-basic-auth
+learn_more: ''
+category: Devtools
+type: 3rd-party
+maintainers:
+  - name: kgierke
+    github: kgierke
+compatibility:
+  nuxt: ^2.0.0
```

we might need to add compatibility with the module options:

```suggestion
  nuxt: ^3.0.0
```